ui/src/plugin/PluginDetailView.tsx

```diff
@@ -0,0 +1,259 @@
+import React, {Component} from 'react';
+import {RouteComponentProps} from 'react-router';
+import ReactMarkDown from 'react-markdown';
+import MonacoEditor from 'react-monaco-editor';
+import axios from 'axios';
+import Info from '@material-ui/icons/Info';
+import Build from '@material-ui/icons/Build';
+import Subject from '@material-ui/icons/Subject';
+import Refresh from '@material-ui/icons/Refresh';
+import Button from '@material-ui/core/Button';
+import DefaultPage from '../common/DefaultPage';
+import * as config from '../config';
+import Container from '../common/Container';
+
+interface IProps extends RouteComponentProps<{id: string}> {}
+
+interface IState {
+    pluginInfo: IPlugin | null;
+    displayText: string | null;
+    currentConfig: string | null;
+}
+
+export default class PluginDetailView extends Component<IProps, IState> {
+    public state: IState = {pluginInfo: null, displayText: null, currentConfig: null};
+
+    public componentDidMount() {
+        const {match} = this.props;
+        axios.get<IPlugin[]>(`${config.get('url')}plugin`).then((response) => {
+            const pluginInfo: IPlugin = response.data.filter(
+                (p) => p.id.toString() === match.params.id
+            )[0];
+            this.setState({pluginInfo});
+            this.refreshFeatures();
+        });
+    }
+
+    private async refreshFeatures() {
```

Comment:
Now both request calls are executed in sequence, but they could been done in parrallel:
```
 private async refreshFeatures(): Promise<void> {
        return Promise.all([this.refreshConfigurer(), this.refreshDisplayer()]).then(() => {});
    }
```